tests/integration/test_vulnerability_detector/data/feeds/canonical/custom_ubuntu22_oval_feed.xml

```diff
@@ -0,0 +1,57 @@
+
+    <generator>
+        <oval:product_name>Canonical CVE OVAL Generator</oval:product_name>
+        <oval:product_version>1.1</oval:product_version>
+        <oval:schema_version>5.11.1</oval:schema_version>
+        <oval:timestamp>2022-05-02T15:47:01</oval:timestamp>
+    </generator>
+
+    <definitions>
+        <definition class="inventory" id="oval:com.ubuntu.jammy:def:100" version="1">
+            <metadata>
+                <title>Check that Ubuntu 22.04 LTS (jammy) is installed.</title>
+                <description></description>
+            </metadata>
+            <criteria>
+                <criterion test_ref="oval:com.ubuntu.jammy:tst:100" comment="The host is part of the unix family." />
+                <criterion test_ref="oval:com.ubuntu.jammy:tst:101" comment="The host is running Ubuntu jammy." />
+            </criteria>
+        </definition>
+    </definitions>
+    <tests>
+        <ind-def:family_test id="oval:com.ubuntu.jammy:tst:100" check="at least one" check_existence="at_least_one_exists" version="1" comment="Is the host part of the unix family?">
+            <ind-def:object object_ref="oval:com.ubuntu.jammy:obj:100"/>
+            <ind-def:state state_ref="oval:com.ubuntu.jammy:ste:100"/>
+        </ind-def:family_test>
+
+        <ind-def:textfilecontent54_test id="oval:com.ubuntu.jammy:tst:101" check="at least one" check_existence="at_least_one_exists" version="1" comment="Is the host running Ubuntu jammy?">
+            <ind-def:object object_ref="oval:com.ubuntu.jammy:obj:101"/>
+            <ind-def:state state_ref="oval:com.ubuntu.jammy:ste:101"/>
+        </ind-def:textfilecontent54_test>
+    </tests>
+    <objects>
+        <ind-def:family_object id="oval:com.ubuntu.jammy:obj:100" version="1" comment="The singleton family object."/>
+
+        <ind-def:textfilecontent54_object id="oval:com.ubuntu.jammy:obj:101" version="1" comment="The singleton release codename object.">
+            <ind-def:filepath>/etc/lsb-release</ind-def:filepath>
+            <ind-def:pattern operation="pattern match">^[\s\S]*DISTRIB_CODENAME=([a-z]+)$</ind-def:pattern>
+            <ind-def:instance datatype="int">1</ind-def:instance>
+        </ind-def:textfilecontent54_object>
+    </objects>
+    <states>
+        <ind-def:family_state id="oval:com.ubuntu.jammy:ste:100" version="1" comment="The singleton family object.">
+            <ind-def:family>unix</ind-def:family>
+        </ind-def:family_state>
+
+        <ind-def:textfilecontent54_state id="oval:com.ubuntu.jammy:ste:101" version="1" comment="Ubuntu 22.04 LTS">
+            <ind-def:subexpression>jammy</ind-def:subexpression>
+        </ind-def:textfilecontent54_state>
+    </states>
+</oval_definitions>
```

Comment:
Why is it added here? this was added on `2836-support-for-ubuntu22-vdt` branch. 

Also, remember that this PR should have the parent  `2836-support-for-ubuntu22-vdt` branch, not the master.

Reply:
Fixed. This file was added on the `2836-support-for-ubuntu22-vdt` branch, where this branch comes from. By pointing the PR at master it thought the file was being added.